src/www/ui_components/server-connection-viz.html

```diff
@@ -167,26 +188,49 @@
         z-index: 400;
       }
 
-      @media (min-height: 600px) {
+      @media (max-width: 360px) {
         #sm,
         #sm-grey {
+          top: 12px;
+          left: 12px;
+          height: 8px;
+          width: 8px;
+        }
+        #md,
+        #md-grey {
+          top: 8px;
+          left: 8px;
+          height: 16px;
+          width: 16px;
+        }
+        #lg,
```

Comment:
Nit: could we use "small", "medium", and "large" instead?

Reply:
Done.